lightning/backend/backend.go

```diff
@@ -388,3 +397,16 @@ type Rows interface {
 	// capacity with the current instance. The typical usage is `x = x.Clear()`.
 	Clear() Rows
 }
+
+type EngineWriter interface {
```

Comment:
How `Engine` communicates with `EngineWriter`. In #493, the disk quota checking routine may trigger force flush and import action if the disk is nearly full, so each engine write should support force flush then